src/Components/bookList.js

```diff
@@ -0,0 +1,22 @@
+import Book from './Book';
+
+const BookList = () => (
+  <>
+    <div className="book-list">
+      <p>Category</p>
+      <div className="book-desc">
+        <Book title="The Hunger Games" author="Suzanne Collins" />
+      </div>
+      <button type="button">Remove</button>
+    </div>
+    <div className="book-list">
+      <p>Category</p>
+      <div className="book-desc">
+        <Book title="Dune" author="Frank Herbert" />
+      </div>
+      <button type="button">Remove</button>
+    </div>
```

Comment:
- [ ] Remove button should be inside the Book component, this way it will be easier to handle functionality in upcoming milestones. Kindly render Remove button from Book component